mappings/net/minecraft/structure/pool/SinglePoolElement.mapping

```diff
@@ -1,8 +1,11 @@
 CLASS net/minecraft/class_3781 net/minecraft/structure/pool/SinglePoolElement
 	FIELD field_16674 processors Lnet/minecraft/class_6880;
 	FIELD field_24015 location Lcom/mojang/datafixers/util/Either;
+	FIELD field_24951 LOCATION_CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24952 REGISTRY_CODEC Lcom/mojang/serialization/Codec;
```

Comment:
This doesn't seem to relate to registries. `REGISTRY_CODEC` is used for `Codec<RegistryEntry<A>>` in current yarn mappings.
```suggestion
	FIELD field_24952 CODEC Lcom/mojang/serialization/Codec;
```